delivery_admin/views/react/orders.jsx

```diff
@@ -18,11 +18,50 @@ class Orders extends React.Component {
     })
   }
 
+  count_orders(orders){
+    var count = {}
+    for(var i in orders){
+      var meal = orders[i].meal
+      count[meal] = count[meal] ? count[meal]+1 : 1
+    }
+    return count
+  }
+
   render () {
+
```

Comment:
No spaces inside a function. Spaces are helpful to quickly identify functions inside the class, if there are other spaces the code becomes less readable at a glance. In case you feel like the render method is too long or difficult to read, break up this component in smaller ones, or see if you can simplify the page (Better). If a component is pretty simple you can[ just use a function](https://facebook.github.io/react/docs/components-and-props.html)